RadixWallet/Features/DappsAndPersonas/AuthorizedDApps/AuthorizedDApps+View.swift

```diff
@@ -10,71 +29,44 @@ extension AuthorizedDappsFeature {
 		public init(store: Store) {
 			self.store = store
 		}
-	}
 
-	public struct ViewState: Equatable {
-		let dApps: IdentifiedArrayOf<Dapp>
-
-		struct Dapp: Equatable, Identifiable {
-			let id: AuthorizedDapp.ID
-			let name: String
-			let thumbnail: URL?
-		}
-	}
-}
+		public var body: some SwiftUI.View {
+			WithViewStore(store, observe: { $0 }) { viewStore in
+				ScrollView {
+					VStack(spacing: .medium1) {
+						Text(L10n.AuthorizedDapps.subtitle)
+							.textBlock
+							.frame(maxWidth: .infinity, alignment: .leading)
```

Comment:
Here's the fix

Reply:
I would put this under the `ScrollView`. Otherwise, if we ever remove or modify this `Text`, is possible that we make the same mistake again.